### ISSUE TITLE
[Keyboard Navigation-Project-Makecode-Documentation]Search icon is not accessible using keyboard showing near Search edit box.

### ISSUE THREAD

**Issue by v-mesala** [Microsoft/pxt#2805]
**User Experience:**
User who depends on Keyboard with Screen Reader may get restricted from accessing all the functionalities if that element is not accessible using keyboard.

**Test Environment**
OS: RS2 Version 1703(OS Build 15063.2)
Platform: Edge.

**Repro Steps**
1- Navigate to https://makecode.microbit.org/projects
2- Navigate to the Search editable box.
3- Verify that Search icon showing near Search box is accessible using keyboard or not.

**Actual Result**
Search icon is not accessible using keyboard showing near Search edit box on project Documentation page

**Expected Result**
Search icon should be accessible using keyboard showing near Search edit box on Project Documentation Page.

**MAS Reference**
https://microsoft.sharepoint.com/teams/msenable/Documents/LCA/MAS%20-%20July%201,%202017%20Release/MAS%202.1.1%20%E2%80%93%20Keyboard.docx?d=w8492c4ebc17940ae8777cd044ed725a2&csf=1

**Refer Attachment**
[MAS2.1.1_MakeCode_Search icon is not accessible.zip](https://github.com/Microsoft/pxt/files/1248840/MAS2.1.1_MakeCode_Search.icon.is.not.accessible.zip)

**Comment by veler** [comment]
This issue is duplicated [https://github.com/Microsoft/pxt/issues/2625](https://github.com/Microsoft/pxt/issues/2625). It has been fixed but is not deployed yet.

**Comment by veler** [comment]
Fix available on [https://makecode.microbit.org/](https://makecode.microbit.org/)

**Comment by v-mesala** [comment]
Verified on URL: https://makecode.microbit.org/projects and issue is fixed now. Hence closing the bug
